### ISSUE TITLE
Tabs in stead of spaces?

### ISSUE THREAD

**Issue by 0phoff** [chuyik/brackets-snippets#37]
All the spaces in the snippets are converted to tabs! It would be nice if the snippets could see the difference between tabs and spaces!

I put tab size on 4 so you can clearly see what i mean.
![image](https://cloud.githubusercontent.com/assets/11853089/7618030/98f2bbe2-f9af-11e4-929b-610da4976c51.png)

**Comment by chuyik** [comment]
Hi dude! The snippets would be converted to tabs if the editor is using tab for indentation.
I know there are some issues about indentation, but would you mind to check your indentation setting (bottom-right corner)?

**Comment by 0phoff** [comment]
yeah i know its because my settings are set to tabs and that was my point.. In my opinion spaces in the snippets need to stay spaces regardless of your indent settings! :)
Dont know anything about making extensions for brackets but maybe you could force the indentation to be tabs in the edit snippet input field. You could then save the snippets with proper tabs and spaces.
If a user has his indent setting to be spaces and the snippets has tabs, the tabs get converted into spaces! :)
I changed my tab settings to spaces (for other reasons..) so i dont really care to much about this issue anymore but might be worth changing for ther people that do use tabs! :)

**Comment by chuyik** [comment]
> spaces in the snippets need to stay spaces regardless of your indent settings

Well some dudes say that snippets should follow the indent settings lol, and I'm working on it.
Maybe I should give an option for it, convert or not.

**Comment by 0phoff** [comment]
yeah of course the snippets should follow your indent settings but the way you are implementing it now is that if your indent setting is set to tabs, every space gets converted to a tab which is not what people want (at least, i imagine its not what they want... :p )
The snippets should know the difference between a space and a tab and display them accordingly (if your settings is set to spaces, the tabs get converted to spaces. But if your indent is tabs, tabs get dipslayed as tabs and spaces as spaces..) But ill let you work on it and see the result afterwards! :D

**Comment by chuyik** [comment]
@0phoff Every space gets converted to a tab? Maybe you should try the new version just released today ^^

**Comment by mcshaman** [comment]
I would also like to see spaces stay as spaces and tabs stay as tabs in the snippets. I prefer tabs for indentation but when I am doing documentation the standard notation for a Doc comment is:

```javascript
/**
 * Documentation here...
 */
```

The leading space in front of each of the star characters is used to make sure all the stars line up for aesthetic reasons and also doc compilers expect this formatting.

As a sub note I am having trouble using snippets for comments. The tab `${1:documentation}` is rendered literally... so I cant tab to it.
